src/event_form/components/calendar/base-calendar.controller.js

```diff
@@ -176,6 +177,10 @@ function BaseCalendarController(calendar, $scope) {
             (timeSpan.start && timeSpan.end) &&
             moment(timeSpan.start).isSame(timeSpan.end, 'day') &&
             moment(timeSpan.start).isAfter(timeSpan.end);
+      },
+      'tooFarInFuture': function (timespan) {
+        var maxDate = moment(new Date()).add(calendar.maxYearTimeSpan, 'y');
```

Comment:
```suggestion
        var maxDate = moment().add(calendar.maxYearTimeSpan, 'y');
```

Reply:
I'm not sure if this is good practice for `momentjs`, but when you create without argument it should return the current date, no?